Formula/packetbeat.rb

```diff
@@ -0,0 +1,55 @@
+class Packetbeat < Formula
+  desc "Lightweight Shipper for Network Data"
+  homepage "https://www.elastic.co/products/beats/packetbeat"
+  url "https://github.com/elastic/beats/archive/v5.0.1.tar.gz"
+  sha256 "707bea8d8eb8cc358bddfcaffa5e90da2a09fab00f2fc53608cc013c79743117"
+
+  head "https://github.com/elastic/beats.git"
+
+  depends_on "go" => :build
+
+  def install
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/elastic/beats").install Dir["{*,.git,.gitignore}"]
+
+    ENV["GOPATH"] = gopath
+
+    cd gopath/"src/github.com/elastic/beats/packetbeat" do
+      system "make"
+      libexec.install "packetbeat"
+
+      inreplace "packetbeat.yml", "packetbeat.interfaces.device: any", "packetbeat.interfaces.device: en0"
```

Comment:
Under OS X `any` is not supported. It has to be `en0` or `lo0`. The binary package of packetbeat shipped for OS X has already `en0` in the config as we do it during packaging. But as we get it here from the source, `any` has to bereplaced by `en0`.